### PR TITLE
Fixes Build summary is incorrect when build with default target

### PR DESCRIPTION
**Pull request by huanlin**
When running a build with default targets, the build summary should be correct now.

### CODE REVIEW

**Comment by mzorec** [comment]
It looks like it's not ok now if you remove the SetAsDefault from target

**Comment by huanlin** [comment]
> It looks like it's not ok now if you remove the SetAsDefault from target

Oops sorry! Hopefuly it's all fixed now.

File: src/FlubuCore/Targeting/TargetTree.cs [hunk 1]
```diff
@@ -288,33 +290,48 @@ public virtual void LogBuildSummary(IFlubuSession session)
...
 
-            int maxTargetNameLength;
-
             if (session.Args.DryRun)
             {
                 session.LogInfo("DRY RUN PERFORMED");
             }
             else if (session.UnknownTarget.HasValue && !session.UnknownTarget.Value)
             {
-                var targets = GetTargetsInExecutionOrder(session);
```
Comment (huanlin) [inline src/FlubuCore/Targeting/TargetTree.cs:299]: Code block from here is refactored to ode block from here is refactored to method `LogBuildSummaryTable`

File: src/FlubuCore/Targeting/TargetTree.cs [hunk 2]
```diff
@@ -288,33 +290,48 @@ public virtual void LogBuildSummary(IFlubuSession session)
...
+            const string DurationTitle = "Duration";
+            const string StatusTitle = "Status";
+            const int DurationLength = 8;
+            const int StatusLength = 10;
+
+            var targetsInOrder = GetTargetsInExecutionOrder(session);
+            if (targetsInOrder.Count < 1)
+            {
+                return;
+            }
```
Comment (huanlin) [inline src/FlubuCore/Targeting/TargetTree.cs:315]: So when there is no targets specified, build summary is skipped.
For example, when showing a built-in task 'help', it won't show the build summary.

File: src/FlubuCore/Targeting/TargetTree.cs [hunk 3]
```diff
@@ -4,8 +4,6 @@
 using System.Drawing;
 #endif
 using System.Linq;
-using System.Reflection.Metadata.Ecma335;
-using System.Runtime.InteropServices.ComTypes;
 using FlubuCore.Context;
```
Comment (huanlin) [inline src/FlubuCore/Targeting/TargetTree.cs]: No idea when and how they got into here. Weird.

**Pull request merged**